vendor/engines/saas/app/jobs/base_login_job.rb

```diff
@@ -0,0 +1,36 @@
+class LoginJob < ActiveJob::Base
+
+  def perform(params)
+    payload = map_payload(params)
+    Analytics::IdentifyUserJob.perform_later(payload)
+
+    #email = map_email(params)
+    #Queue job to send the email
+  end
+
+  def self.action(action)
+    @action = action
+  end
```

Comment:
We use this pattern in the healthchecks

https://github.com/huboard/huboard-web/blob/master/lib/health_checking/health_checks/health_check.rb#L5-L40

I remember there was a reason for it